### DISCUSSION TITLE
Nitrux ISO with MAUI Shell?

### DISCUSSION THREAD

**Discussion by Thatoo** [Nitrux/nitrux-bug-tracker#164] [Q&A] [answered]
Is it possible to install Nitrux with Maui Shell instead of KDE Plasma?

**Comment by UriHerrera** [comment, marked as answer]
This doesn't seem like an issue; I will move this to a discussion.

We have not recently released an ISO with *only* Maui Shell. 

You can already use Maui Shell in the distribution. You can use Maui Shell as an AppImage, available from its repository. Alternatively, you can use a container and compile it (although the `master` branch is being ported to Qt6),

**Reply by Vistaus** [reply]
A new testing ISO would be great! Even if only to test in distro form, so that you can prepare for a stable ISO with Maui Shell later this year.
